### ISSUE TITLE
Permission Denied in nginx container

### ISSUE THREAD

**Issue by gitkent** [hjacobs/kube-resource-report#146]
Hi guys,

I have the following after deployed into a Kubernetes cluster:

 └─ $ ▶ kubectl logs kube-resource-report-np-797b5fbb8d-l75xp -c nginx
2020/04/15 02:38:08 [emerg] 1#1: bind() to 0.0.0.0:80 failed (13: Permission denied)
nginx: [emerg] bind() to 0.0.0.0:80 failed (13: Permission denied)

Can someone please help?

**Comment by jsenon** [comment]
it's due to the security policy applied to the nginx container, that are not able to bind the port `        securityContext:
          runAsNonRoot: true
          runAsUser: 1000`

**Comment by gitkent** [comment]
https://github.com/hjacobs/kube-resource-report/blob/5a95098aaafe48a73fa9d11387ffcf4eb3ceb491/chart/kube-resource-report/values.yaml#L55

It looks like port 80 can't be bound by non-root user. Changing port number to any number >1000 seems to work.

```
container:
  port: 8888
  portName: http
  portProtocol: TCP
```

**Comment by gitkent** [comment]
Fixed by #165
